scm/src/GFS_typedefs.F90

```diff
@@ -3289,15 +3265,7 @@ subroutine control_initialize (Model, nlunit, fn_nml, me, master, &
     endif
     Model%levrp1           = Model%levr + 1
     Model%nfxr             = nfxr
-    Model%aero_in          = aero_in
```

Comment:
This statement was never updated to use iaerclm. It is only set conditionally on line 3279 below. Adding Model%iaerclm = iaerclm before line 3278 solves my seg faults.